pca/db/database_model.py

```diff
@@ -417,3 +419,38 @@ def save(self, *args, **kwargs):
         else:
             self.internal_ip_int = None  # IPv6 integer representations are too large to store in MongoDB currently
         super(ApplicationDoc, self).save(*args, **kwargs)
+
+class UserReportDoc(RootDoc):
+    """Provides schema for User Report doc.
+    customer - a customer ID that corresponds to a CustomerDoc
+    assessment - an assessment ID that corresponds to an AssessmentDoc
+    campaign - a campaign ID that corresponds to a CampaignDoc
+    first_report - the timestamp when the first user click ("report") was received during a phishing campaign
+    total_num_reports - the total number of user clicks ("reports") for a campaign
+    """
+    customer = fields.ReferenceField(CustomerDoc, required=True)
+    assessment = fields.ReferenceField(AssessmentDoc, required=True)
+    campaign = fields.ReferenceField(CampaignDoc, required=True)
+    first_report = fields.DateTimeField(blank=True)
+    total_num_reports = fields.IntegerField(blank=False)
+
+    class Meta:
+        """Meta class for UserReportDoc."""
+
+        collection_name = USER_REPORT_COLLECTION
+        final = True
+        ignore_unknown_fields = True  # TODO see if this can be inherited from RootDoc
```

Comment:
You should create an issue for this TODO, and include a link to the issue in this comment.

Reply:
Issue can be found here: [Issue](https://github.com/cisagov/pca-core/issues/7)

Reply:
@JCantu248, still remains to be done.  Re-request my review after creating the GitHub issue and referencing it in this comment and I'll approve.

Reply:
Umm okay.  #7

Reply:
See commit abaedb5bdfa316bbf3ba6c0d93c873b56b431977 for code changes